pwa-devdocs/_drafts/talons/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Talons
+---
+
+<!-- TODO: Publish this topic under src/peregrine -->
+
+Most [React component][] contain two distinct sections: a section for logic and a presentational section.
+
+The logic section contains code for generating new values from props, maintaining local state, and lifecycle or side effect methods.
+This section can be considered the "brains" of the component.
+
+The presentational section of a React component contains code that define the component's DOM structure.
+It often uses values from the logic part of the component to pass down into another component or display using HTML elements.
+
+PWA Studio separates these two sections into distinct components with specific concerns: a Venia UI component and a Peregrine Talon.
+
+## Peregrine talon overview
+
+Peregrine talons are the logic component counterparts for Venia UI components.
+Each talon in Peregrine is designed to provide data or perform side effects for a specific Venia UI component.
+
+For example, the [`SearchBar`][] component uses the [`useSearchBar()`][] talon to get initial values and functions it needs to handle changes in focus or search term value.
+The `useSearchBar()` talon also keeps track of the component's `expanded` state, which determines how the search bar looks.
+
+## Talons versus hooks
+
+Talons are a PWA Studio concept that represent a specialized type of React hook.
+
+Peregrine **hooks**, such as `useQuery()` and `useWindowSize()`, are designed to be re-usable pieces of code, but
```

Comment:
`useQuery` is being deprecated/removed. Use a different one like `useDropdown` maybe.